src/mol-model-props/computed/accessible-surface-area/shrake-rupley.ts

```diff
@@ -89,12 +89,19 @@ namespace AccessibleSurfaceArea {
         return points;
     }
 
-    export const enum Flag {
+    export const enum Flags {
         NA = 0x0,
         Buried = 0x1,
         Accessible = 0x2
     }
 
+    export const Flag = {
```

Comment:
For this pattern, shall we always define the `enum` in term of the `const enum`? Also we should be consistent with the use of the plural for either the `enum`or the `const enum`.